README.md

```diff
@@ -85,13 +85,13 @@ kube-bench node --version 1.13
 ```
 
 `controls` for the various versions of Kubernetes can be found in directories
-with same name as the Kubernetes versions under `cfg/`, for example `cfg/1.13`.
+with the same name as the Kubernetes versions under `cfg/`, for example `cfg/1.13`.
 `controls` are also organized by distribution under the `cfg` directory for
 example `cfg/ocp-3.10`.
 
 ### Running inside a container
 
-You can avoid installing kube-bench on the host by running it inside a container using the host PID namespace and mounting the `/etc` and `/var` directories where the configuration and other files are located on the host, so that kube-bench can check their existence and permissions. 
+You can avoid installing kube-bench on the host by running it inside a container using the host PID namespace and mounting the `/etc` and `/var` directories where the configuration and other files are located on the host so that kube-bench can check their existence and permissions. 
```

Comment:
```suggestion
You can avoid installing kube-bench on the host by running it inside a container using the host PID namespace and mounting the `/etc` and `/var` directories where the configuration and other files are located on the host, so that kube-bench can check their existence and permissions. 
```